src/ui/public/js/govukFrontend.js.LICENSE.txt

```diff
@@ -77,6 +77,12 @@
  * @preserve
  */
 
+/**
```

Comment:
was this done automatically?

Reply:
Yes, it's populated by GOV design library.